images/stackrox-test.Dockerfile

```diff
@@ -35,14 +38,16 @@ RUN yum update -y && \
         kubectl \
         lsof \
         openssl \
-        @postgresql:12 \
         procps \
         unzip \
         xz \
         zip \
+        lz4 \
```

Comment:
nit: the other packages are sorted alphabetically. Can this be moved after `lsof`?